pkg/git/git.go

```diff
@@ -1,25 +1,104 @@
 package git
 
 import (
+	"encoding/json"
+	"fmt"
+	"net/url"
 	"os/exec"
+	"strings"
 )
 
+var Path = ""
```

Comment:
nit: could this be passed on as function argument instead of a global variable, as it would be hard to track the value of a global function if down the line multiple callers are interacting with this package